pkg/controller/cloneset/sync/cloneset_scale.go

```diff
@@ -172,9 +172,9 @@ func (r *realControl) managePreparingDelete(cs *appsv1alpha1.CloneSet, pods, pod
 			continue
 		}
 
-		klog.V(3).Infof("CloneSet %s patch pod %s lifecycle from PreparingDelete to Normal",
+		klog.V(3).Infof("CloneSet %s patch pod %s lifecycle from PreparingDelete to Creating",
```

Comment:
Creating ->PreNormal

Actually since the users cancel the deletions of pods, we should logging explicitly that 

"CloneSet %s cancel deletion of pod %"